src/main.rs

```diff
@@ -149,16 +151,11 @@ async fn test_timeout(timeout: u64) {
 }
 
 #[async_std::main]
-async fn main() {
-    let config = get_config(env::args().nth(1).unwrap());
-    if config.is_err() {
-        eprintln!("{:?}", config.err().unwrap());
-        exit(1);
-    }
-    let config = config.unwrap();
+async fn main() -> Result<()> {
+    let config = get_config(env::args().nth(1).unwrap())?;
```

Comment:
this `.unwrap()` is intentional?

Reply:
`.nth(1)` returns an `Option`, so I couldn't just question-mark it. That being said, this one probably warrants an `.ok_or()`. Fixing that now.